### PR TITLE
Service name > Service ID in service create form

### PR DESCRIPTION
**Pull request by MatApple**
Changed copy from `SERVICE NAME` to `SERVICE ID` which now aligns with the config section (REVIEW & RUN).

### CODE REVIEW

File: plugins/services/src/js/components/forms/GeneralServiceFormSection.js [hunk 1]
```diff
@@ -302,7 +302,7 @@ class GeneralServiceFormSection extends Component {
           Services
         </h2>
         <p>
-          Configure your service below. Start by giving your service a name.
+          Configure your service below. Start by giving your service an Id.
```
Comment (mlunoe) [inline plugins/services/src/js/components/forms/GeneralServiceFormSection.js]: Let's use ID (uppercase) in the paragraph as well

Comment (MatApple) [inline plugins/services/src/js/components/forms/GeneralServiceFormSection.js]: I disagree - It looks strange with ID. I did try that first. @leemunroe @ashenden anyone want to weigh in here?

Comment (ashenden) [inline plugins/services/src/js/components/forms/GeneralServiceFormSection.js]: It should be All Caps - http://english.stackexchange.com/questions/101248/how-should-the-abbreviation-for-identifier-be-capitalized

**Comment by MatApple** [comment]
@mlunoe @ashenden thanks - updated the copy.

**Comment by mlunoe** [comment]
Let's ⛵️ this thing :)

**Pull request merged**